recipes/flock/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: flock
+  version: "1.0"
+source:
+  fn: 12b153db8d1e0192c449296d9348d2f33d279293.tar.gz
+  url: https://github.com/cristhomas/immport-test/archive/12b153db8d1e0192c449296d9348d2f33d279293.tar.gz
+  md5: 0d4821d92847d0f9376249bec7ce8ee8
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
+  run:
+    - libgcc    # [not osx]
```

Comment:
Unfortunately the author did not provide a --help or --version or equivalent. Can I add a test file?

What am I missing from the guidelines?